src/html/htmprint.cpp

```diff
@@ -709,7 +711,10 @@ bool wxHtmlEasyPrinting::DoPrint(wxHtmlPrintout *printout)
     wxPrintDialogData printDialogData(*GetPrintData());
     wxPrinter printer(&printDialogData);
 
-    if (!printer.Print(m_ParentWindow, printout, true))
+    bool dialog = m_dialog != DIALOG_NEVER;
+    if (m_dialog == DIALOG_ONCE) m_dialog = DIALOG_NEVER;
```

Comment:
Some more nitpicking - the assignment statement should not be at the same line as the `if` expression.

Reply:
Yes, these are good points, thanks.

Reply:
Fixed.